internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -27,6 +27,10 @@ type DynamicScaleUpManager struct {
 }
 
 var _ workers.Worker = &DynamicScaleUpManager{}
+var clusterStatesTowardReadyState = []string{
```

Comment:
This has been added as a global variable but not used in dynamic scale up mgr.

I also wonder if it would be a good idea to just duplicate this to not affect both dynamic scaleup/scaledown mgrs if the set of states changes

Reply:
an IDE refactoring not completely finished. 
I should have remove the local reference.

I am fine with duplicating it either way as I think the value won't change

Reply:
done